.github/workflows/test.yml

```diff
@@ -44,13 +44,14 @@ jobs:
     - name: Generate code coverage report from .xcresult/
       run: ./scripts/generate-code-coverage-report.sh 
   
-    - name: Upload code coverage report 
-      uses: codecov/codecov-action@v2
-      with: 
-        token: ${{ secrets.CODECOV_UPLOAD_TOKEN }} # not required for public repos, but sometimes uploads fail without it so include it anyway
-        fail_ci_if_error: true # fail if upload fails so we can catch it and fix it right away.
-        verbose: true 
-        directory: .build/generated
+      # Disable uploading code coverage for now until we investigate the 500 errors we're getting from Codecov.
+    # - name: Upload code coverage report 
+    #   uses: codecov/codecov-action@v2
+    #   with: 
+    #     token: ${{ secrets.CODECOV_UPLOAD_TOKEN }} # not required for public repos, but sometimes uploads fail without it so include it anyway
+    #     fail_ci_if_error: true # fail if upload fails so we can catch it and fix it right away.
+    #     verbose: true 
+    #     directory: .build/generated
```

Comment:
[Draft PR made that is trying to fix this](https://github.com/customerio/customerio-ios/pull/324)